include/boost

```diff
@@ -0,0 +1 @@
+maidsafe
```

Comment:
:)

That's a symbolic link on POSIX, and it causes b2 headers to map the crux headers in as boost/crux. I had been hoping that Windows git would do something sensible with symbolic links, but I was wrong. I guess it can be deleted, I'll do that later.